src/helpers/elements-interactions.ts

```diff
@@ -9,6 +10,22 @@ async function waitUntilElementDisappear(page: Page, elementSelector: string, ti
   await page.waitForSelector(elementSelector, { hidden: true, timeout });
 }
 
+async function waitUntilIframeFound(page: Page, framePredicate: (frame: Frame) => boolean, timeout = 30000) {
+  let frame: Frame | undefined;
+  await waitUntil(() => {
+    frame = page
+      .frames()
+      .find(framePredicate);
+    return Promise.resolve(!!frame);
+  }, 'wait for iframe with login form', timeout, 1000);
```

Comment:
consider getting the description from arg or use generic message instead like `wait for iframe`

Reply:
Done